nbgrader/html/templates/student_submissions.tpl

```diff
@@ -27,9 +27,9 @@
   {%- for submission in submissions -%}
   <tr>
     <td>
-      <a href="/submissions/{{ submission.id }}">
+      {%- if notebook_server_exists -%}<a href="/submissions/{{ submission.id }}">{%- endif -%}
```

Comment:
Same thing here -- otherwise you can't open the student's submission for grading at all.